src/ray/protobuf/common.proto

```diff
@@ -589,6 +589,13 @@ message TaskInfoEntry {
   // If the task/actor is created within a placement group,
   // this value is configured.
   optional bytes placement_group_id = 26;
+  // Tasks arguments passed in as (object) references.
+  //
+  // NOTE: This list only contains `ObjectReference`s passed in as arguments
+  //       this task is dependent on and does NOT contain
+  //        - Args passed by value (inlined)
+  //        - ObjectRefs of the args passed by value
+  repeated ObjectReference dependent_args_refs = 27;
```

Comment:
I think we can only record the ObjectID instead of the entire ObjectRef?

Reply:
Surely, but why not entire `ObjectRef`?

Reply:
I think we only care about ObjectID so trying to reduce memory usage: we need to store lots of events in GCS also each task can potentially have unlimited number of args.